docs/src/modules/java/pages/key-value-entities.adoc

```diff
@@ -67,7 +67,8 @@ include::example$key-value-counter/src/main/java/com/example/application/Counter
 ----
 <1> Every Entity must be annotated with `@ComponentId` with a stable unique identifier for this entity type.
 <2> The `CounterEntity` class should extend `akka.javasdk.keyvalueentity.KeyValueEntity`.
-<3> The initial state of each counter is defined with value 0.
+<3> Stores the `entityId` on an internal attribute so we can use it later. Alternatively, if inside a command handler, `commandContext().entityId()` also provides such information.
```

Comment:
In the context of a KV is less useful to store the id in a field and you usually have it available when you need anyway (command handler).. in any case, since the `KeyValueContext` can also be used to provide more information (e.g. `selfRegion`) about the entity, I think it's probably good to have it here explicitly..